docs/cmd_list.txt

```diff
@@ -8,32 +8,30 @@ Options:
   --help                          Show this message and exit.
 
 Configuration commands:
-  ping  Health check REANA server.
+  ping  Check connection to REANA server.
 
 Workflow management commands:
-  create  Create a REANA compatible workflow from REANA specifications file.
-  delete  Delete a workflow run.
-  diff    Show differences between two workflows.
-  list    List all available workflows.
+  create  Create a new analysis workflow.
```

Comment:
"Create a new workflow"

here and elsewhere, if we choose to hide "run", then let's also hide "analysis" for consistency, so we can simply speak of creating workflows, running and stopping workflows, etc, without the word "analysis" anywhere